### PR TITLE
Made changes to make getAssets faster

### PR DESCRIPTION
**Pull request by rangav**
- Removed filename and mediaType props to make getAssets faster
- Created variable fetchResults to initialize on first access or startFrom = 0
- Set Image Loader priority so no need to unlink cameraRoll library ( removes extra step )

### CODE REVIEW

File: ios/RNGalleryManager.m [hunk 1]
```diff
@@ -130,19 +138,10 @@ @implementation RNGalleryManager
...
-    NSString *uit = ((PHAssetResource*)resources[0]).uniformTypeIdentifier;
-    NSString *mimeType = (NSString *)CFBridgingRelease(UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassMIMEType));
-    CFStringRef extension = UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassFilenameExtension);
-    
     [assets addObject:@{
                         @"type": [self getMediaType:([asset mediaType])],
                         @"width": @([asset pixelWidth]),
                         @"height": @([asset pixelHeight]),
-                        @"filename": orgFilename ?: @"",
-                        @"mimeType": mimeType ?: @"",
```
Comment (pentarex) [inline ios/RNGalleryManager.m:145]: My application depends on the mime type...

File: ios/RNGalleryManager.m [hunk 2]
```diff
@@ -130,19 +138,10 @@ @implementation RNGalleryManager
...
                         @"id": [asset localIdentifier],
                         @"creationDate": [asset creationDate],
                         @"uri": [self buildAssetUri:[asset localIdentifier] extension:extension],
```
Comment (pentarex) [inline ios/RNGalleryManager.m:147]: where is the extension coming from?

**Review by pentarex — CHANGES_REQUESTED** [review]
Please check the comments that Ive applied.

File: ios/RNGalleryManager.m (hunk 1 above)
Comment (rangav) [reply ios/RNGalleryManager.m:145]: ok you can keep mimeType then.

File: ios/RNGalleryManager.m (hunk 2 above)
Comment (rangav) [reply ios/RNGalleryManager.m:147]: sorry forgot about the uri. 

if you don't need extension i will make these additional changes

In RNGalleryManager.m 145
```@"uri": [NSString stringWithFormat:@"ph://%@", [asset localIdentifier]]```

in RctImageLoader.m
```
NSString *assetID = [imageURL.absoluteString substringFromIndex:@"ph://".length];
results = [PHAsset fetchAssetsWithLocalIdentifiers:@[assetID] options:nil];
```

This is what default RN loader is using.

if you are happy i will make these changes

**Pull request closed without merging**